src/compiler/emitter.ts

```diff
@@ -2274,6 +2286,20 @@ namespace ts {
                     write(",");
                 }
 
+
+                // Emit any trailing comment of the last element in the list
+                // i.e
+                //       var array = [...
+                //          2
+                //          /* end of element 2 */
+                //       ];
+                if (previousSibling && delimiter && previousSibling.end !== parentNode.end) {
+                    emitLeadingComments(previousSibling.end, /*isEmittedNode*/ previousSibling.kind !== SyntaxKind.NotEmittedStatement);
+                    if (hasTrailingComma) {

```

Comment:
Why are you emitting this twice? Also, statements are never emitted with delimiters, so your `isEmittedNode` argument will always be `true`.